spec/active_force/active_query_spec.rb

```diff
@@ -40,6 +39,70 @@
     end
   end
 
+  describe '#blank? delegation' do
+    before do
+      allow(client).to receive(:query).and_return(api_result)
+    end
+
+    context 'when there are no records' do
+      let(:api_result) { [] }
+
+      it 'returns true' do
+        result = active_query.where("Text_Label = 'foo'").blank?
+        expect(result).to be true
+      end
+
+      it 'returns true' do
+        result = active_query.where("Text_Label = 'foo'").first.blank?
```

Comment:
this is not utilizing the delegation. this spec is basically confirming that you can call `.first` on an ActiveQuery object, find nil, and then call `.blank?` on that nil. you could comment out the delegator in the ActiveQuery model and this spec will still pass